tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -390,6 +390,86 @@ Object.Pipeline {
 			}
 		}
 	}
+
+	gain-capture.5 {
+		format		$FORMAT
+		index		18
+		Object.Widget.pipeline.1 {
+			stream_name "DMIC0 Reference"
+		}
+		Object.Widget.copier.1 {
+			stream_name "DMIC0 Reference"
```

Comment:
Reference is heavily tainted by 'echo reference'.

Usually for capture we use 'Raw'. That's also a Windows term, see https://techcommunity.microsoft.com/t5/discussions/raw-audio-capture-flag/m-p/3048056

Reply:
sure, thanks